src/plugins/kibana/server/lib/schemas/resources/index_pattern_schema.js

```diff
@@ -6,6 +6,7 @@ module.exports = Joi.object({
   time_field_name: Joi.string(),
   interval_name: Joi.string(),
   not_expandable: Joi.boolean(),
+  fieldFilters: Joi.string().default('[]'),
```

Comment:
To stay consistent with the way the other optional fields are handled, I don't think we should set a default here. An index pattern could be created in a number of ways (including a user creating it directly in ES) so we can't rely on this default anyway.